crates/orderbook-client/src/lib.rs

```diff
@@ -95,7 +93,7 @@ async fn subscribe_impl(
                     };
 
                     match msg {
-                        tungstenite::Message::Pong(_) => {
+                        tungstenite::Message::Text(s) if s.eq_ignore_ascii_case("pong") => {
```

Comment:
This is the aforementioned Pong -> "pong" change which _might_ be breaking (?)

Reply:
I don't think it should be as it doesn't seem that we ever actually use Ping/Pong with the orderbook